### PR TITLE
remove fast_dot which is deprecated in sklearn

### PR DESCRIPTION
**Pull request by agramfort**
fast_dot is deprecated from sklearn

### CODE REVIEW

File: mne/viz/utils.py [hunk 1]
```diff
@@ -108,7 +108,8 @@ def tight_layout(pad=1.2, h_pad=None, w_pad=None, fig=None):
                 warn(msg)
     except Exception:
         try:
-            fig.set_tight_layout(dict(pad=pad, h_pad=h_pad, w_pad=w_pad))
+            with catch_warnings(record=True) as ws:
+                fig.set_tight_layout(dict(pad=pad, h_pad=h_pad, w_pad=w_pad))
```
Comment (Eric89GXL) [inline mne/viz/utils.py]: should probably add the same conditional that is used above, no?

**Comment by agramfort** [comment]
ok for @dengemann ?

**Comment by dengemann** [comment]
Yes!

**Comment by agramfort** [comment]
ok let's merge when CIs are happy

**Comment by Eric89GXL** [comment]
Thanks @agramfort

**Pull request merged**